x/nftmarket/keeper/nft_listing.go

```diff
@@ -183,6 +231,9 @@ func (k Keeper) ListNft(ctx sdk.Context, msg *types.MsgListNft) error {
 		return types.ErrNftListingAlreadyExists
 	}
 
+	// todo:delete
+	k.TestMint(ctx, msg.Sender.AccAddress(), msg.NftId.ClassId, msg.NftId.NftId)
```

Comment:
These test code will need to be removed.

Reply:
ok

Reply:
I fixed